waku/node/waku_node.nim

```diff
@@ -1239,7 +1251,7 @@ proc mountLibp2pPing*(node: WakuNode) {.async: (raises: []).} =
 
 # TODO: Move this logic to PeerManager
 proc keepaliveLoop(node: WakuNode, keepalive: chronos.Duration) {.async.} =
-  while node.started:
+  while true:
```

Comment:
Wouldn't this fail if the node has not started? I wonder if it should be like this:
```
while true:
   if not node.started: await sleepAsync(keepalive)
`` 
so it does not execute the keep alive loop unless the node has actually started?

Reply:
Ooh great catch! Will add that logic :))